src/components/shared/map/MapCenter.tsx

```diff
@@ -64,57 +64,47 @@ const MapCenter = () => {
     ) : null;
   };
 
+  // Function to save the map center directly to the backend
   const handleSetCenter = async () => {
     if (center !== null && currentUser?.pi_uid) {
-      console.log(center);
-      localStorage.setItem('mapCenter', JSON.stringify([center.lat, center.lng]));
+      console.log(center); // Log the center coordinates for debugging
 
       try {
-        const response = await saveMapCenter(center.lat, center.lng);
-        console.log('Map center saved successfully', response);
-        setShowPopup(true);
+        const response = await saveMapCenter(center.lat, center.lng); // Save the map center to the backend
+        console.log('Map center saved successfully', response); // Log success response
+        setShowPopup(true); // Show the confirmation popup
       } catch (error) {
-        console.error('Error saving map center:', error);
+        console.error('Error saving map center:', error); // Log any errors during the save process
       }
     } else {
-      console.log('Center or currentUser.pi_uid is null');
+      console.log('User not authenticated or center coordinates are null'); // Handle the case where the user is not authenticated or coordinates are null
     }
   };
 
+  // Function to handle the closing of the confirmation popup
   const handleClickDialog = () => {
     setShowPopup(false);
   };
 
-  // define map boundaries
```

Comment:
Not sure why you're removing the map constraints.  These are put into place to fix the map boundaries and tiling issue.